stripe-core/src/main/java/com/stripe/android/core/networking/RequestExecutor.kt

```diff
@@ -88,3 +88,36 @@ suspend fun <Response : StripeModel> executeRequestWithResultParser(
         }
     )
 }
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP_PREFIX)
+suspend fun executeRequestWithErrorParsing(
```

Comment:
I think you can use `executeRequestWithModelJsonParser`, and if parsed to `Unit`, just pass `Unit.serializer`

Reply:
I had tried that before but `Unit` is not a `StripeModel`, and `serializer()` does not return a `ModelJsonParser`